mobile-center-link-scripts/src/ios/MobileCenterConfig.js

```diff
@@ -40,7 +40,15 @@ function addConfigToProject(file) {
     return new Promise(function (resolve, reject) {
         debug(`Trying to add ${file} to XCode project`);
 
-        var globString = `**/${(pjson && pjson.name ? pjson.name : '*')}.xcodeproj/project.pbxproj`
+        var globString;
+        if (pjson && pjson.name) {
+            globString = `**/${(pjson.name.replace(/-/g, ''))}`;
+        }
+        else {
+            globString = `ios/*`;
```

Comment:
nitpick: use single quote instead of backtick when not using ES6 string interpolation.